.docker/data/start.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+echo "===== START OF BACKEND ====="
+cd /home/airline/airline/airline-data
+SBT_OPTS="-Xmx4G -Xms512M -XX:MaxMetaspaceSize=512M" sbt "runMain com.patson.MainSimulation"
```

Comment:
should be an envvar, but 1G from the default is definitely not enough, it makes the backend crash with a heap exception shortly after startup